### PR TITLE
Update PR Template to Include New Description/Overview Guidelines

### PR DESCRIPTION
**Pull request by ShaunaGordon**
### Stakeholder Overview

We're in the process of improving the PR template to not only be more useful in writing PRs, but also in reading them. In the future, this will allow us to use them as the basis for automated deployment notifications, changelog aggregation, and other archival and communication purposes.

### Changes

- Change "description" to "Stakeholder Overview" to make section purpose more explicit and self-documenting
- Add more elaborate helper notes to encourage more desired descriptions

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,8 +1,25 @@
...
+Good example:
+IHP is in the process of updating the way carts get built. This is the core cart creation 
+logic that sorts and groups orders into carts. With the upcoming change for the UI, users will 
+be able to skip the Press Staging Queue and create a cart based on the flash count for the 
+printer they are selecting for.
+
+Good example:
+We are currently sending all pageViewTags as custom events to fullstory which results in Fullstory 
+throttling Custom Events for our accounts. This prevents us from using custom events for more legitimate 
+users (e.g. custom event to track when a user is in test group)
```
Comment (chmurph2) [inline .github/pull_request_template.md]:
```suggestion
users (e.g. custom event to track when a user is in test group).
```
Add missing period.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,8 +1,25 @@
...
+This change is going to help Swag Management to identify sessions of users in Swag Management test group.
```
Comment (chmurph2) [inline .github/pull_request_template.md]:
```suggestion
This change is going to help Swag Management to identify sessions of users in the Swag Management test group.
```

**Review by chmurph2 — APPROVED** [review]
LGTM, fixed some typos. The "good examples" seem good. 🙂

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,8 +1,25 @@
-### Description
+### Stakeholder Overview
 
 <!--
-Describe the relevant motivation and context for this change.
-Please include a summary of the change as well as the issue that is fixed.
+This section should provide a brief overview of the changes and the motivation for the changes in a 
+manner friendly to non-technical people and to technical people who don't know the context or are not
+subject matter experts on the system or update being made. Consider the logical need for the change, 
+as well as the logical effect of the change, particularly as any end-users would experience it.
```
Comment (invacuo) [inline .github/pull_request_template.md]: I was gonna suggest some minor edits here but then I InkGPT-ed it.


```suggestion
Provide a concise summary of the motivation and the driving force behind them. These explanations are designed to be understandable to individuals without technical knowledge, and to people who may not be familiar with the context or specialize in the particular system or update being discussed. Think about the logical rationale for the modification and its practical impact, especially from the viewpoint of any end-users.
```

Or here is another one

```suggestion
Provide a clear and concise summary of what changes are being made and the reasons for these adjustments. The explanation should be easy to understand for both non-tech savvy individuals and technical personnel who may not be well-acquainted with the context of the changes or may not be experts in the system being updated. Give thought to the logical necessity for the changes, as well as the likely consequences these changes could bring about, especially from the perspective of potential end-users.
```

**Review by invacuo — APPROVED** [review]
Just left some InkGPT suggestions to make it more concise. Totally fine if you prefer to leave it as it is too. Thanks for taking ownership and pushing this forward 👏

**Review by kgillison — APPROVED** [review]
I like it ❤️

**Pull request merged**